src/web/Content/client/dumpling.py

```diff
@@ -944,6 +953,24 @@ def _add_key_if_not_exists(dictProp, key, val):
         if not key in dictProp:
             dictProp[key] = val
 
+    @staticmethod
+    def _create_hang_dump(pid, outpath, debuggerpath):
+        osStr = platform.system().lower()
+        command = ""
+        if osStr == 'linux':
+            command = "./" + debuggerpath + " " + pid + " --name " + outpath + "\memdum" + pid + ".dmp"
+        elif osStr == 'windows':
+            outputpath = outpath + "\memdum" + pid + ".dmp"        
+            command = debuggerpath + " -p "+ pid + " -c " + '".dump /ma '+outputpath+';.detach;q"'
+        else:
+            return
+
+        print "creating dump"
+        status = os.system(command)
+        
+        if status == 1:
```

Comment:
As part of the verification of `debuggerpath` we should probably make sure that we are writing out an actionable error to the console.